Apps/W1/ShopifyConnector/app/Integration/Tables/ShpfyRegisteredStore.Table.al

```diff
@@ -0,0 +1,48 @@
+/// <summary>
+/// Table Shpfy Registered Store (ID 30136).
+/// </summary>
+table 30136 "Shpfy Registered Store"
+{
+    Access = Internal;
+    Caption = 'Registered Store';
+    DataClassification = SystemMetadata;
+
+    fields
+    {
+        field(1; Store; Text[50])
+        {
+            Caption = 'Store';
+            DataClassification = SystemMetadata;
+        }
+        field(2; RequestedScope; Text[1024])
```

Comment:
please rename field to: "Requested Scope"

Reply:
Updated with commit "Object Renaming"